### DISCUSSION TITLE
Wyze Refresh time?

### DISCUSSION THREAD

**Discussion by DanielRSims** [jfarmer08/homebridge-wyze-smart-home#246] [General]
Are there any plans to allow you to put the slider at 0 or at least 1000 for Refresh time in Milliseconds? Right now you can only go as low as 60 seconds. I want the devices to update right away. as it is now when I open a door, it takes a whole minute to update. Makes the plugin sortof useless when creating automations. Anyone find a fix? or know if there are plans to make this quicker?

**Comment by jfarmer08** [comment]
I don't plan on making this any quicker by default. You can manually change the config, but keep in mind that you can get locked out by polling too often.
